### PR TITLE
bugfix: [Blog] Fix Blog Tags Responsiveness and 

### LINKED ISSUE #1551: [Blog] Missing image and tags responsiveness
### Description

- [x] In [meshery/blog](https://meshery.io/blog/page/2/) the image is missing for the blog.
- [ ] The tags are not looking good for smaller screens

### Screenshots
Screenshot for both missing image and responsiveness issue of tags
Screenshot 2023-12-21 181430

### Expected Behavior
The images should be visible along with the tags to have flex-column for smaller screens.
image

### Enviroment:
 - OS: Windows
 - Browser: Chrome 
 - Version: 
 - Device: Desktop & Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by horizenight**
**Description**

This PR fixes #1551

**Notes for Reviewers**
As the CSS names were all connected , solved related issues also.

[bugFix : [Blog] Overflowing Callout Card Border](https://github.com/meshery/meshery.io/commit/f1e0f2b89b414ada1f6431947a7eaac7a4ad52df) commit solves the overflow on [page](https://meshery.io/blog/unleashing-power-and-flexibility-the-synergy-of-rest-and-graphql-in-meshery)
image

[bugFix : [Blog] Breaking CodeHighlight in Blog Article](https://github.com/meshery/meshery.io/commit/cdf81ee5bdc09a665731ee702ff4a94ceeaafb32) solves the breakage on this [page](https://meshery.io/blog/meshery-v06-release)
image

[bugFix : [Blog] Fix Wierd Tag and Title OverFlow in Blog Articles](https://github.com/meshery/meshery.io/commit/99b2841e74540c023d65fdc6e4fefadc6c8ebbb9) solves the main tags issue 

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
@horizenight The text size of tags is too small for mobile screens. Please check the design I have provided in the issue and try replicating it.
image

File: _sass/blog.scss [hunk 1]
```diff
@@ -139,6 +139,40 @@
...
+    font-size: 0.6em;
+  }
+  
+  .blog-post h3 a {
+    font-size: 1em;
+  }
+  .page h2{
+    font-size: 1.5em;
+  }
+}
```
Comment (ashutosh-rath02) [inline _sass/blog.scss:175]: Instead try using `flex-column` and `flex-direction: column-reverse` for  `.blog-post .post-details` and `align-items: flex-start` for ` .post-date`  at smaller screens.

File: _sass/navigation.scss [hunk 2]
```diff
@@ -235,6 +235,17 @@ a.nav-text:hover {
...
+	.highlight{
+		padding: .25rem 0.75rem;
+	  }
+	  code {
+		font-size: 0.6em;
+	  }
+}
+@media screen and (max-width: 370px) {
+
+}
```
Comment (ashutosh-rath02) [inline _sass/navigation.scss:248]: You can remove these.

File: _sass/navigation.scss [hunk 3]
```diff
@@ -235,6 +235,17 @@ a.nav-text:hover {
 	}
 }
 
+@media screen and (max-width: 520px) {
+	.highlight{
+		padding: .25rem 0.75rem;
+	  }
+	  code {
+		font-size: 0.6em;
```
Comment (ashutosh-rath02) [inline _sass/navigation.scss:243]: Changing font-size should not be an option, please work this out.

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

**Comment by abhijeetgauravm** [comment]
Hey @horizenight Let's discuss this on website call today at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by horizenight** [comment]
Hello @abhijeetgauravm 
I am travelling a lot this week, so it would be great if you have some design guidelines that will work on that : ) 
when available 👍🏼

**Comment by leecalcote** [comment]
@horizenight are you still working on this?

**Comment by leecalcote** [comment]
If not, @ashutosh-rath02, will you finish this off?

**Comment by leecalcote** [comment]
@horizenight We'll consider this PR abandoned, yes?

**Comment by sudhanshutech** [comment]
@ashutosh-rath02 as you are taking care of this area and there is no response from @horizenight , we can close this one.
Thanks @horizenight will get you another issue when you are active

**Pull request closed without merging**